specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
@kiniry I think this would be an excellent idea because it would help us narrow down what exactly needs to be implemented as Java classes; in particular, @jldodds and I are experiencing some confusion at the moment as we try to implement `CastVoteRecord` and `Vote`.